nucypher/blockchain/eth/actors.py

```diff
@@ -131,23 +131,23 @@ def __init__(self, registry: BaseContractRegistry, **kwargs):
         super().__init__(registry=registry, **kwargs)
         self.__token_agent = None
 
-    @property
-    def token_agent(self):
-        if self.__token_agent:
-            return self.__token_agent
-        self.__token_agent = ContractAgency.get_agent(
-            NucypherTokenAgent,
-            provider_uri=self.eth_provider_uri,
-            registry=self.registry,
-        )
-        return self.__token_agent
-
-    @property
-    def token_balance(self) -> NU:
-        """Return this actor's current token balance"""
-        balance = int(self.token_agent.get_balance(address=self.checksum_address))
-        nu_balance = NU(balance, 'NuNit')
-        return nu_balance
+    # @property
+    # def token_agent(self):
+    #     if self.__token_agent:
+    #         return self.__token_agent
+    #     self.__token_agent = ContractAgency.get_agent(
+    #         NucypherTokenAgent,
+    #         blockchain_endpoint=self.eth_provider_uri,    # TODO this is fishy - and doesn't seem needed?
+    #         registry=self.registry,
+    #     )
+    #     return self.__token_agent
+    #
+    # @property
+    # def token_balance(self) -> NU:
+    #     """Return this actor's current token balance"""
+    #     balance = int(self.token_agent.get_balance(address=self.checksum_address))
+    #     nu_balance = NU(balance, 'NuNit')
+    #     return nu_balance
```

Comment:
I need to have a closer look, but this is most likely an artifact from legacy staking tools.

Reply:
✔️